fullcontact/api/base/base.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module serves the base class for all
+FullContact API request classes.
+"""
+from abc import ABCMeta
+from typing import Callable, TypeVar
+
+from requests import Response
+
+from ...__about__ import __version__
+from ...config.client_config import ClientConfig
+from ...exceptions import FullContactException
+from ...response.base.base import BaseApiResponse
+from ...schema.base.schema_base import BaseSchema
+
+BaseResponse = TypeVar("BaseResponse", bound=BaseApiResponse)
+
+
+class ApiBase(object, metaclass=ABCMeta):
```

Comment:
Let's have more consistent name in API class. Add `Api` to all API classes at the end. This can be `BaseApi`.

Reply:
Ok, I'll chang it.

Reply:
Done 👍